File: TP2DPBO2025C2/php/Kode Program/Aksesoris.php
<?php
require "Petshop.php";

class Aksesoris extends Petshop {
    private $jenis;
    private $bahan;
    private $warna;

    public function __construct($ID, $Nama, $Kategori, $Harga, $Gambar, $jenis, $bahan, $warna) {
        parent::__construct($ID, $Nama, $Kategori, $Harga, $Gambar);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    public function getjenis() {
        return $this->jenis;
    }

    public function getbahan() {
        return $this->bahan;
    }

    public function getwarna() {
        return $this->warna;
    }

    public function setbahan($bahan) {
        $this->bahan = $bahan;
    }

    public function setwarna($warna) {
        $this->warna = $warna;
    }

    public function setjenis($jenis) {
        $this->jenis = $jenis;
    }

    public function __destruct() {
    }

}
?>

File: TP2DPBO2025C2/Python/Kode Program/Aksesoris.py
from Petshop import Petshop 

class Aksesoris(Petshop):

    def __init__(self,ID, Nama, Kategori, Harga, jenis, bahan, warna):
        super().__init__(ID, Nama, Kategori, Harga)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna

    def setJenis(self, jenis):
        self.jenis = jenis

    def setBahan(self, bahan):
        self.bahan = bahan

    def setWarna(self, warna):
        self.warna = warna

    def getJenis(self):
        return self.jenis

    def getBahan(self):
        return self.bahan

    def getWarna(self):
        return self.warna
